_episodes/08-development-setup.md

```diff
@@ -171,6 +176,14 @@ If the installation is successful, ESMValTool prints a help message to the conso
 > {: .solution}
 {: .challenge}
 
+### 4 Updating ESMValTool
+
+The ``master`` branch has the latest features of ESMValTool. Please make sure
+that the source code on your machine is up-to-date. If you obtain the source
+code using git clone as explained in step **1 Source code**, you can run ``git pull``
+to update the source code. Then ESMValTool installation will be updated
+with changes from the ``master`` branch.
+
```

Comment:
A few things here : They have to run git pull from their local master branch and then switch to their working branch to do a merge if they want to update their local branch. This should be made clear I think. Should we give the commands for that? Also, we need to say that if they previously downloaded the code the only way they can update is to redownload the zip file and create the environment afresh.

Reply:
Thank you for your comment. I agree that this information is very helpful. I am afraid if the lesson should cover this information. I would like to focus on teaching `esmvaltool development and contribution` rather than teaching `Git`. At the beginning of this lesson, it is mentioned that users need Git knowledge for this episode. The git pull, merge and changing branches are all covered in Git documentation and it is beyond the scope of this lesson. The same applies to downloading the source code.

Reply:
@SarahAlidoost , Thanks! I thought of that and I do agree. I want to record this discussion in our archives with this final thought if I may : If a user is not familiar with Github, there are other ways to install/use ESMvalTool and they can pick up the development mode with Github eventually after knowing more about how to use GitHub. So it is ok to not include git commands to update code here.